automation/test.sh

```diff
@@ -2,10 +2,16 @@
 
 kubectl() { cluster/kubectl.sh "$@"; }
 
+teardown() {
+    kubectl get --all-namespaces event || true
+    kubectl get --all-namespaces pod || true
```

Comment:
this would be madness on openshift, too many pods. are we interested in anything but nmstate namespace?

Reply:
Let's have it all, you never know what you need and this is just after finishing it.

Reply:
(*demonic voice* you will regret it JAJAJA)

ack